airlock/templates/update_files.html

```diff
@@ -0,0 +1,37 @@
+{# hide default button, as we are going to autoshow the modal anyway #}
+{% fragment as button %}
+{% endfragment %}
```

Comment:
Pretty sure that the above lines can be deleted, and the line below can change to 
```
{% #modal id="updateRequestFile" autoshow=True %}
```

Reply:
I think this is because it's still sharing some template stuff with the add_files dialogue - maybe I should do a further refactor of that. I'll probably need to touch that anyway when re-enabling updating multiple files.

Reply:
I'm going to merge & do this in my next PR, hope that's ok.